src/crimson/os/seastore/seastore_types.h

```diff
@@ -517,6 +520,14 @@ struct omap_root_t {
   depth_t get_depth() const {
     return depth;
   }
+
+  void set_hint(laddr_t addr_min) {
+    hint = addr_min;
+  }
```

Comment:
Is it possible to remove this setter and set the hint through `omap_root_le_t::get(hint) -> omap_root_t` instead ?

Reply:
done.